ReactNativeImageCropping.m

```diff
@@ -88,8 +88,24 @@ - (NSDictionary *)constantsToExport
 
 - (void)handleImageLoad:(UIImage *)image {
     
+    // hardcoded downsample of image
```

Comment:
I think it's a good feature to have especially if delay is noticeable. Did you think about JS API already?

So far we have:

`.cropImageWithUrlAndAspect(imageUrl, aspectRatio)`
`.cropImageWithUrl(originalImage.uri)`

without breaking backwards compatibility we could introduce something like
`.cropImage(originalImage.uri, options)`
where options would be:
```
{
    maxWidth: 150, // optional, default no max width
    aspectRatio: ReactNativeImageCropping.AspectRatioSquare, // optional, default not locked
}
```

It would be easy to extend as well.